netbout-web/src/test/java/com/netbout/rest/MarkdownTest.java

```diff
@@ -162,6 +162,20 @@ public void formatsBulletsToHtml() throws Exception {
     }
 
     /**
+     * Markdown can break a single line.
+     * @throws Exception If there is some problem inside
```

Comment:
@antonini @krzyk we have the paper http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html

>It's a good practice to always declare test methods as throwing Exception.

I think we should follow the paper.